File: ITP1/2/B/B.kt
fun main(args: Array<String>) {
  val line = readLine()!!.split(' ').map{x->x.toInt()}
  val a    = line[0]
  val b    = line[1]
  val c    = line[2]

  val result = if (a < b && b < c) {
    "Yes"
  } else {
    "No"
  }
  println("${result}")
}

File: ITP1/2/B/B.go
package main

import (
	"bufio"
	. "fmt"
	"os"
	"strconv"
)

var sc = bufio.NewScanner(os.Stdin)

func init() {
	sc.Split(bufio.ScanWords)
}

func nextInt() int {
	sc.Scan()
	i, e := strconv.Atoi(sc.Text())
	if e != nil {
		panic(e)
	}
	return i
}

func main() {
	a, b, c := nextInt(), nextInt(), nextInt()

	if a < b && b < c {
		Println("Yes")
	} else {
		Println("No")
	}
}
